Release 2.0.0

ekohl opened the pull request:
This changes the Rakefile to not use the v prefix in tags. This is to stay consistent with all existing tags.

ekohl committed:
Release 2.0.0

alexjfisher commented on Rakefile:
```diff
@@ -7,8 +7,7 @@ task :default => :spec
   GitHubChangelogGenerator::RakeTask.new :changelog do |config|
-    version = '1.0.0'
-    config.future_release = "v#{version}" if version =~ /^\d+\.\d+.\d+$/
+    config.future_release = '2.0.0'
```
Is it worth moving the version number to a single file like we do in some other gems?
eg https://github.com/voxpupuli/hiera-eyaml-gpg/blob/master/lib/hiera/backend/eyaml/encryptors/gpg/version.rb

ekohl replied:
Different implementation, same result. I now load the version from the gemspec which keeps the code simpler.

ekohl committed:
Load the version from the gemspec

ekohl committed:
Drop explicit mime-types dependency

This isn't used within the project. In the past it was there to pin it
for Ruby 1.8 support but now it doesn't add anything.

ekohl committed:
Drop Ruby 1.8 compatibility guard
